TF_Settings_Web/src/Pages/Camera/Masking/sceneRendering.ts

```diff
@@ -72,6 +72,11 @@ const dummy = new Object3D();
 const BASE_LINE_THICKNESS = 0.005;
 
 export const setupRenderScene = (div: HTMLDivElement) => {
+    if (div.children.length > 0) {
```

Comment:
💬 Why is this needed?

Reply:
It's not! It'll be removed

Reply:
Could a comment be added as for why that block of code is needed

Reply:
Added!